app/components/Swapbox.tsx

```diff
@@ -211,6 +214,10 @@ export const Swapbox = ({ market }: { market: FixedProductMarketMaker }) => {
 
   const outcomeList = [outcome0, outcome1];
 
+  const unsupportedNetwork =
+    !!connectorChainId &&
+    !supportedChains.some(supportedChain => supportedChain.id === connectorChainId);
+
```

Comment:
We could abstract this function because we're also using it in `app/components/NetworkButton.tsx`

Reply:
I feel like this one doesn't need abstraction since it's being used in 2 places. If we need it 1 more time lets abstract it.